argusclient/client.py

```diff
@@ -391,7 +391,18 @@ def add(self, alert):
             alertobj.trigger = alertobj.triggers.add(alert.trigger)
         if alert.notification:
             alertobj.notification = alertobj.notifications.add(alert.notification)
-        if alert.trigger and alert.notification:
+        elif alert.notifications and isinstance(alert.notifications, list):
+            notifications = []
+            for notification in alert.notifications:
+                notifications.append(alertobj.notifications.add(notification))
+            alertobj.notifications = notifications
+        if alert.trigger and alert.notifications:
+            alertobj.trigger.notificationsIds = []
+            for notification in alertobj.notifications:
+                self.argus.alerts.add_notification_trigger(alertobj.id, notification.id, alertobj.trigger.id)
```

Comment:
Looks like this automatically connects notifications, but only when there is a single trigger. Could you add some additional documentation to this method on this behavior?